python/datasources/data_source.py

```diff
@@ -42,27 +42,28 @@ def write_to_bq_table(self, dataset: str, gcs_bucket: str, filename: str, table_
         gcs_bucket: The name of the gcs bucket to read the data from
         filename: The name of the file in the gcs bucket to read from
         table_name: The name of the BigQuery table to write to"""
-        frame = gcs_to_bq_util.load_csv_as_dataframe(gcs_bucket, filename)
+        chunked_frame = gcs_to_bq_util.load_csv_as_dataframe(
```

Comment:
Actually wait. For sources that override this method, some of them may be larger than the chunk size added to the util file but wont iterate over the chunks to append them all to bq.

Reply:
Oh wow good catch. Added an additional parameter so that data frames don't come back unexpectedly chunked.